packages/block/package.json

```diff
@@ -16,7 +16,7 @@
   },
   "license": "MPL-2.0",
   "author": "mjbecze (mb@ethdev.com)",
-  "type": "commonjs",
+  "type": "module",
```

Comment:
Why is this changed here?  This property is set to `commonjs` in the other package level `package.json` files and overridden to `module` in the ESM build step.

Reply:
Vitest will run, but give a warning about deprecating their commonjs support.
changing this in all the package.json files is a fix for this, but we can't do that to every package yet.

Reply:
Yeah, agree with Andrew that we should handle such a change with very much care and give this some thought (likely Andrew best to discuss this), since it could affect our final builds + internal tooling.

Reply:
I don't think we should make this change anywhere for now.  I know our configs are not perfect but we should be able to set up a separate package.json just for vitest that specifies module if that's what we want to do longer term.